scripts/forc-update.ts

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
```

Comment:
The objective of this file is to be executed and update the `forc-bin` version in accordance with the source `forc bin`.

I think it could be implemented inside the `forc-bin` folder also. But ignore it on dist.